File: Client/src/app/modules/management/models/teacher.model.ts
export interface ITeacher {
    id: number;
    name: string;
    surname: string;
    patronymic: string;
    universityId: number;
}

export interface ISaveTeacher {
    name: string;
    surname: string;
    patronymic: string;
    universityId: number;
}

File: Services/TeacherService/TeacherService/Models/TeacherModel.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Business.Models
{
    public class TeacherModel : BaseModel
    {
        public string Name { get; set; }
        public string Surname { get; set; }
        public string Patronymic { get; set; }
    }
}
